engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/repository-details-manage-ostree-branches.controller.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * @ngdoc object
+ * @name  Bastion.repositories.controller:RepositoryManageContentController
+ *
+ * @requires $scope
+ * @requires translate
+ * @requires Repository
+ *
+ * @description
+ *   Provides the functionality for the repository details pane.
+ */
+angular.module('Bastion.repositories').controller('RepositoryManageOstreeBranchesController',
+    ['$scope', 'translate', 'Repository',
+    function ($scope, translate, Repository) {
+
+        function resetBranches() {
+            $scope.branch = {
+                editMode: false,
+                working: false
+            };
+        }
+
+        function saveBranches(branches, success, errors) {
+            $scope.repository["ostree_branches"] = branches;
+            $scope.repository.$update(function (repository) {
+                $scope.successMessages = [translate('Repository Branches Updated.')];
+                $scope.wrap(repository);
+                if (success) {
+                    success(repository);
+                }
+            }, function (response) {
+                _.each(response.data.errors, function (errorMessage) {
+                    $scope.errorMessages.push(translate("An error occurred saving the Repository: ") + errorMessage);
+                });
+                if (errors) {
+                    errors(response);
+                }
+            });
```

Comment:
In functions below, you defined success and error functions and then passed those to the call at the end which is nice for readability. I am OK either way at this point, but wanted to point that out if you wanted to adjust. ACK overall either way for me.